### PR TITLE
feat: added babel with minified and polyfill in core

### PR DESCRIPTION
**Pull request by hailem**

### CODE REVIEW

File: packages/core/.babelrc.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  presets: [
+    ['@babel/preset-env', {modules: false, targets: {node: 'current'}}],
```
Comment (sonic182) [inline packages/core/.babelrc.js:3]: I have a tiny different babel config file

https://github.com/doofinder/webcomponents/blob/johanderson/smartsample/babel.config.json


and I think you miss core-js dependency for polyfills -> https://babeljs.io/docs/en/babel-preset-env#usebuiltins

Importing once core-js and the runtime stuff they suggest in the NOTE section there

File: packages/core/.babelrc.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    ['@babel/preset-typescript'],
```
Comment (sonic182) [inline packages/core/.babelrc.js:4]: I think this isn't needed (rollup handle typescrypt)

File: packages/core/package.json [hunk 3]
```diff
@@ -10,24 +10,34 @@
   "types": "lib/core.d.ts",
   "repository": "git@github.com:doofinder/webcomponents.git",
   "author": "Doofinder Devs <https://github.com/doofinder>",
+  "browserslist": [
+    "defaults"
+  ],
   "scripts": {
     "build": "yarn compile && yarn typings && rimraf types temp",
+    "babel": "babel './src/index.ts' -d lib2",
```
Comment (sonic182) [inline packages/core/package.json:18]: what is that? the `-d lib2` stuff

**Comment by carlosescri** [comment]
@hailem fix conflicts please

**Pull request merged**